frontend/src/Constants.js

```diff
@@ -74,6 +74,7 @@ export const managerReportStatuses = [
 
 export const REPORT_STATUSES = {
   DRAFT: 'draft',
+  DELETED: 'deleted',
```

Comment:
Was included in previous backend PR but for some reason it wasn't merged into the codebase? /shrug